src/main/java/org/openmaptiles/layers/Landuse.java

```diff
@@ -103,10 +109,36 @@ public void process(Tables.OsmLandusePolygon element, FeatureCollector features)
       if ("grave_yard".equals(clazz)) {
         clazz = FieldValues.CLASS_CEMETERY;
       }
-      features.polygon(LAYER_NAME).setBufferPixels(BUFFER_SIZE)
+      var feature = features.polygon(LAYER_NAME).setBufferPixels(BUFFER_SIZE)
         .setAttr(Fields.CLASS, clazz)
-        .setMinPixelSizeOverrides(MIN_PIXEL_SIZE_THRESHOLDS)
         .setMinZoom(Z6_CLASSES.contains(clazz) ? 6 : 9);
+      if (FieldValues.CLASS_RESIDENTIAL.equals(clazz)) {
+        feature
+          .setMinPixelSize(0.1)
+          .setPixelTolerance(0.25);
```

Comment:
Where did the min pixel tolerance come from? Ideally we'd simplify the output geometry after merging but it looks like only mergeLineStrings has a resimplify option, we'd need to add it for merge polygons.

Reply:
Short answer: Simply copied from Buildings.

Long answer: This was one of the earliest changes I did so following "_this should work same as for buildings_" I simply copied values from Buildings, since with `MIN_PIXEL_SIZE_THRESHOLDS` merging did not work.

In the mean time, we've also discussed similar issue here: https://github.com/openmaptiles/planetiler-openmaptiles/pull/50#discussion_r1031438065 . Hence if those values are for buildings at Z13, they might not be ideal for landuse at Z6-Z12.

This and then that simplifying after merging vs. my current understanding of handling of geometries, I'm out of my depth and I could use you help to handle it better.

Reply:
OK, the main thing is that the output looks similar to what openmaptiles is producing. The one issue that bigger min pixel sizes could cause is that they filter-out small polygons from lower zoom levels that would end up getting merged but there's likely not many small residential polygons so if the output looks the same then it should be fine.